index.html

```diff
@@ -0,0 +1,177 @@
+
+              <hr class="hr-nav">
+              <span class="link"><a href="#Sponsor">Sponsors</a></span>
+              <hr class="hr-nav">
+              <span class="link"><a href="#News">News</a></span>
+              <hr class="hr-nav">
+              <span class="link"><a href="#NFTintiative">NFT Initiative</a></span>
+              
+            </li>
+          </ul>
+        </div>
+
+        <div class="mobile-menu">
+          <i class="fa-solid fa-bars"></i>
+        </div>
+                  
+        <div class="first-section">
+          <h2 class="med-heading">
+            "Welcome to the Age of NFT's"
+          </h2>
+          <h1 class="main-heading">The 1st Annual <br> Nft Global Summit <br> 2022</h1>
+          <p class="paragraph1">An Expository event unravelling the Best Ideas, Best Builders, best Coverage and the Best community which covers over
+            100 countries taking place in July, in United States</p>
+          <h2 class="date">17(WED) - 29.07.2022(FRI)</h2>
+          <h4 class="venue">@ National Museum of National History, National Mall in Washington, D.C, United States</h4>
+        </div>
+      </section>
+
+      <!-- the Main Program Section -->
+      <section id="Program" class="program">
+        <div class="program-h2">
+          <h2>Main Program</h2>
+          <div class="line"> </div>
+        </div>
+        <div class="container">
+
+          <div class="card1 cards">
+            <i class="fa fa-desktop screen-icon" aria-hidden="true"></i>
+            <h3>Lecture</h3>
+            <p>Listen to the speakers from various countries about the Creating and selling NFTs</p>
+          </div>
+
+          <div class="card2 cards">
+            <i class="fa-solid fa-coins"></i>
+            <h3>NFT Exhibition</h3>
+            <p>Explore different NFTs already trending in the global market</p>
+          </div>
+
+          <div class="card3 cards">
+            <i class="fa-solid fa-comment-dollar"></i>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts and opinions with experts for each topic</p>
+          </div>
+
+          <div class="card4 cards">
+            <i class="fa-solid fa-chalkboard-user"></i>
+            <h3>Workshop</h3>
+            <p>try creating your own NFTs using our various open-end frameworks available</p>
+          </div>
+
+          <div class="card5 cards">
+            <i class="fa-solid fa-link"></i>
+            <h3>NFT Initate</h3>
+            <p>Get opportunities to link up and network with NFT traders and High class buyers all around the world,</p>
+          </div>
+
+        </div>
+        <h2 class="program-text desktop-only">View the Entire Program</h2>
+        <button type = "submit" class="program-text mobile-only">Join NFT Summit 2022</button>
+        
+      </section>
+
+      <!-- The Speakers Section -->
+      <div id="speaker-section"></div>
```

Comment:
- [ ] [REQUIRED] Please make changes!
- 1-Add `horizontal line` : `#EC5242` under the main `h` tag.
- 2-Add `duplicate background(E.G: Chess Bg)` to the speaker's pictures.
- 3-Add `horizontal line` : `#D3D3D3` under the `text: 'MINGs Music Enterprises LLC'` and others.

![2022-07-26_22-51-12](https://user-images.githubusercontent.com/63915024/181078200-f1f73d85-3f9e-4aff-864d-779587c38bdd.png)

Reply:
Hi @mukhammadrizooff 
I wrote in my pull request that "the checkerboard image was not added based on personal taste" of which it tallies with the capstone requirements that I can make changes to image and colors etc but you insisted that I must add it


Please re-review this
however I will make changes to the margin between the title and paragraph as stated in review 3
but for others, I strongly disagree